api/app/signals/auth/tokens.py

```diff
@@ -0,0 +1,57 @@
+from json import loads
+
+from jwcrypto.jws import InvalidJWSSignature
+from jwcrypto.jwt import JWT, JWTExpired, JWTMissingKey
+from rest_framework.exceptions import AuthenticationFailed
+
+from .config import get_settings
+from .jwks import get_keyset
+
+
+class JWTAccessToken():
+    @staticmethod  # noqa: C901
```

Comment:
Maybe we can better instantiate an instance of JWTAccessToken where we need it, instead of using static methods? Or just expose functions?